app/controllers/programs/program_managers_controller.rb

```diff
@@ -1,10 +1,68 @@
-class Programs::ProgramManagersController < ProgramManagersController
+class Programs::ProgramManagersController < ApplicationController
   before_action :set_program_manager_program
 
-  private
+  def new
+    @program_manager = ProgramManager.new
+  end
+
+  def create
```

Comment:
Short  circuiting a conditional is hard to understand. Try creating  a module that will handle the various conditions instead via methods

Reply:
I added comments to make the conditionals clear. I would love to create a module, but I don't know how to do that. I tried to put that code somewhere - nothing worked. 
Could you please teach me how to do that?
Could we do that in a separate ticket?

Reply:
Fixing this in this PR is the basis of PR code reviews. We can fix this in this PR. I will send you an invite for early next week to look at how to create a module to capture this sort of stuff.
In tne meantime look at [this link](https://medium.com/rubycademy/modules-in-ruby-part-i-a2cdfaccdb6e)

Reply:
Fixing this in this PR is the basis of PR code reviews. We can fix this in this PR. I will send you an invite for early next week to look at how to create a module to capture this sort of stuff.
In tne meantime look at [this link](https://medium.com/rubycademy/modules-in-ruby-part-i-a2cdfaccdb6e)

Reply:
@rsmoke I removed this code from the pr to do that in a separate ticket.

Reply:
Resolving this conversation. The code in question was removed for another commit.